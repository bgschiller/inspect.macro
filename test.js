const pluginTester = require('babel-plugin-tester');
const plugin = require('babel-plugin-macros');

pluginTester({
  plugin,
  snapshot: true,
  babelOptions: { filename: __filename },
  tests: [
    `
    const inspect = require('./inspect.macro');

    inspect(1 + 1 * 5, func(14));
    `,
  ],
});