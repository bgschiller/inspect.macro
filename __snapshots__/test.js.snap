// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`macros 1. macros: 1. macros 1`] = `
"
const inspect = require('./inspect.macro');

inspect(1 + 1 * 5, func(14));

      ↓ ↓ ↓ ↓ ↓ ↓

console.log('1 + 1 * 5: ', function () {
  try {
    return 1 + 1 * 5;
  } catch (e) {
    return '(an error occurred)';
  }
}(), 'func(14): ', function () {
  try {
    return func(14);
  } catch (e) {
    return '(an error occurred)';
  }
}());
"
`;
